chanzuckerberg/single-cell-data-portal
# feat: Deletion of published collections in the Curation UX must be removed (#2099)

## Reason for Change

- #2099 

## Changes

- Removed "Delete" button and associated "Delete" modal from Collection detail page.

## Testing steps

- A curator should not be able to delete a published collection.
- A curator should still be able to delete a private collection, or delete a collection in revision.

## Notes for Reviewer

- Tests are unaffected by the removal of the delete button.
